### PR TITLE
Add Atlabs module

### PR DESCRIPTION
**Pull request by mezlet**
Add all endpoints to module
Add tests
Add documentation

### CODE REVIEW

File: src/Atlabs/Common/ModuleRequest.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                'path' => $data["path"],
+                'method' => $data['method'],
+                'requestData' => $requestData
+                ]);
+
+                if(gettype($this->response) === 'array'){
+                    return $this->response;
+                }
+                
+                return $this->response->getBody()->getContents();
```
Comment (akinjide) [inline src/Atlabs/Common/ModuleRequest.php]: Indent

File: src/Atlabs/README.md [hunk 2]
```diff
@@ -0,0 +1,385 @@
+# INNOVATION SANDBOX
+
+## Install
+The recommended way to install innovation-sandbox is through Composer.
+
+```bash
+$ composer require enyata/innovation-sandbox
+```
+
```
Comment (akinjide) [inline src/Atlabs/README.md]: Please sync with @addempsea regarding documentation standard in node package

File: tests/Atlabs/AirtimeTest.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<?php
+
+require_once './tests/Fixtures/Atlabs.php';
```
Comment (akinjide) [inline tests/Atlabs/AirtimeTest.php]: rename `Fixtures -> mock`

**Review by akinjide — CHANGES_REQUESTED** [review]

File: src/Atlabs/Common/ModuleRequest.php (hunk 1 above)
Comment (mezlet) [reply src/Atlabs/Common/ModuleRequest.php]: done

File: src/Atlabs/README.md (hunk 2 above)
Comment (mezlet) [reply src/Atlabs/README.md]: noted

File: tests/Atlabs/AirtimeTest.php (hunk 3 above)
Comment (mezlet) [reply tests/Atlabs/AirtimeTest.php]: done

**Review by akinjide — CHANGES_REQUESTED** [review]
Inconsistent use of single-quote and double-quote string.

**Review by akinjide — APPROVED** [review]

**Pull request merged**